packages/lesswrong/server/repos/helpers.ts

```diff
@@ -15,3 +22,11 @@ export const getViewablePostsSelector = (postsTableAlias?: string) => {
     ${aliasPrefix}"postedAt" IS NOT NULL
   `;
 };
+
+export const getViewableTagsSelector = (tagsTableAlias?: string) => {
```

Comment:
What are the aliases for? They don't seem to be used

Reply:
You would use this if you were filtering tags in a join or subquery, so that the name of the tag is something other than `"Tags"`.